nyc_taxis/challenges/default.json

```diff
@@ -104,7 +123,26 @@
         {
           "name": "refresh-after-index",
           "operation": "refresh"
-        }
+        }{%- if force_merge is defined or force_merge_max_num_segments is defined %},
```

Comment:
I didn't use the [exists_set_param filter](https://github.com/elastic/rally/commit/8d328de2c322a9217fb91627340b125539a6a0d7) to easy compatibility with older Rally versions.